mm2src/rpc.rs

```diff
@@ -38,6 +38,7 @@ mod dispatcher_legacy;
 
 #[path = "rpc/dispatcher/dispatcher_v2.rs"] mod dispatcher_v2;
 
+#[path = "rpc/lp_basis.rs"] mod lp_basis;
```

Comment:
Could you please rename it to `get_public_key`? I hope it also encourage us to separate code to a smaller modules in the future :slightly_smiling_face: